pkg/scanners/kubernetes/parser/parser.go

```diff
@@ -91,7 +91,10 @@ func (p *Parser) required(fs fs.FS, path string) bool {
 		return false
 	}
 	defer func() { _ = f.Close() }()
-	return detection.IsType(path, f, detection.FileTypeKubernetes)
+	if data, err := ioutil.ReadAll(f); err == nil {
```

Comment:
I think we can pass `f` to the `IsType` function without this middle reader?

Reply:
f doesn't implement`io.ReaderSeeker` which is expected by `IsType`